plugins/modules/elastic_pkgpolicy.py

```diff
@@ -77,29 +84,50 @@ def main():
       results['changed'] = False
       module.exit_json(**results)
     
-    if module.params.get('integration_name'):
-      integration_object = kibana.check_integration(integration_name)
+    if module.params.get('integration_title'):
+      integration_object = kibana.check_integration(integration_title)
     else:
       results['integration_status'] = "No Integration Name provided to get the integration object"
       results['changed'] = False
       module.exit_json(**results)
     
-    if not integration_object:
-      results['integration_status'] = 'Integration name is not a valid'
+    if ( integration_name and integration_ver and integration_name) and not integration_object:
```

Comment:
looks like `integration_name` is duplicated here by mistake.

Reply:
I have updated to correct it to say title